index.html

```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
             </ul>
             </div>
            </div>
-              </section> 
+              </section>
+              <section class="contact-area">
+                <div class="contact-me">
+                   <h2 class="contact-title">Contact Me</h2>
+                   <p class="contact-desc">
+                       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
```

Comment:
[Optional] It looks great, but it's important to follow the Figma template. So please try to follow the same Figma font size.

![image](https://user-images.githubusercontent.com/34779928/192621868-4b187a5a-9f9f-46b8-82b5-5f7929acb817.png)